src/web/app/pages-admin/admin-accounts-page/admin-accounts-page.component.ts

```diff
@@ -44,18 +18,20 @@ interface AccountInfo {
 })
 export class AdminAccountsPageComponent implements OnInit {
 
-  instructorCourses: CourseAttributes[] = [];
-  studentCourses: CourseAttributes[] = [];
-  accountInfo: AccountAttributes = {
+  instructorCourses: Course[] = [];
+  studentCourses: Course[] = [];
+  accountInfo: Account = {
     googleId: '',
     name: '',
     email: '',
     isInstructor: false,
+    institute: '',
+    createdAtTimeStamp: 0,
   };
 
   constructor(private route: ActivatedRoute, private router: Router, private httpRequestService: HttpRequestService,
       private navigationService: NavigationService, private statusMessageService: StatusMessageService,
-              private accountService: AccountService) { }
+              private accountService: AccountService, private courseService: CourseService) { }
```

Comment:
Since we're here, let's make this indentation clearer?
```suggestion
constructor(private route: ActivateRoute,
                  private router: Router,
                  private httpRequestServe: HttpRequestService,
                  private navigationService: NavigationService,
                  private statusMessageService: StatusMessageService,
                  private accountService: AccountService,
                  private courseService: CourseService) { }
```